usaspending_api/etl/transaction_loaders/fpds_loader.py

```diff
@@ -65,6 +65,11 @@ def delete_stale_fpds(detached_award_procurement_ids):
         deleted_awards = cursor.fetchall()
         logger.info(f"{len(deleted_awards):,} awards were unlinked from transactions due to pending deletes")
 
+        cursor.execute(
```

Comment:
Not sure if it matters, the added lines to delete from `transaction_search` lead the other delete statements which doesn't follow the current order of string with the "base" table